build/resources/extra/secretshare_deploy.yaml

```diff
@@ -26,7 +25,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: secretshare
-  namespace: ibm-common-services
```

Comment:
@mikekaczmarski Why do we need to remove the namespace here

Reply:
maybe a rebase conflict?

Reply:
Yes, looks like an oversight.  I will re-do